### PR TITLE
script for generating runt.sh and runt.sh itself

### PR DESCRIPTION
**Pull request by mattdurham**
#### PR Description

These are scripts to generate the tutorial script and the tutorial script itself. Split the PR into two parts, this one concerns the scripts and the second will implement those changes in the tutorials itself, along with general cleanup of tutorials and images from the UI. 

This is also linux/osx specific, at some point might do a powershell version for windows.

### CODE REVIEW

**Review by karengermond — APPROVED** [review]
LGTM

File: docs/sources/flow/tutorials/assets/generate.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```
Comment (jcreixell) [inline docs/sources/flow/tutorials/assets/generate.sh:1]: not super important, but you may want to use `#!/usr/bin/env bash` for better portability and consistency with other scripts in this repo

File: docs/sources/flow/tutorials/assets/generate.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# This script generates runt.sh script that pulls down the needed files and runs them.
```
Comment (rfratto) [inline docs/sources/flow/tutorials/assets/generate.sh:3]: I think this comment needs to explain what's going on here a little more. None of us are bash experts, so I find it a little hard to follow. 

I think explaining what the "needed files" are would go a long way here.

I think we should also comment when this script would need to be re-run.

**Review by rfratto — APPROVED** [review]
The approval here is a bit rubber-stampy. If the scripts lint, that's good enough for me, but I find the generate.sh hard to understand and would like to see more comments to explain what's going on so we know what correct output looks like.
